### PR TITLE
Added an element Iron

### PR DESCRIPTION
**Pull request by gaurabga**
I saw that iron was missing in your periodic table! I added some info.

- [x] Added Iron info.

### CODE REVIEW

File: elements/iron.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Symbol = Fe
+mass = 55.845 
+Atomic Number: **26**. 
+
+It is a metal that belongs to the first transition series 
+and of the periodic table. Its electronic configuration is 
+[Ar] 3d^6 4s^2. It is by mass the most common element on 
+Earth, forming much of Earth's outer and inner core. Iron 
+is present in oxygen-transsporting protein hemoglobin in
```
Comment (herbertludowieg) [inline elements/iron.md]: typo in oxygen-transporting

**Comment by rwalkerlewis** [comment]
Very nice, but perhaps we ought to add about where the symbol Fe is from.

File: elements/iron.md (hunk 1 above)
Comment (gaurabga) [reply elements/iron.md]: thanks bro for comment. corrected now!

**Comment by gaurabga** [comment]
> Very nice, but perhaps we ought to add about where the symbol Fe is from.

Thanks for the suggestion. Info added.

**Comment by janash** [comment]
Nice team work, everyone!

**Pull request merged**